tests/foreman/ui/test_classparameters.py

```diff
@@ -253,15 +370,34 @@ def test_positive_update_parameter_type(self):
             3.  Enter a 'valid' default Value.
             4.  Submit the changes.
 
-        :expectedresults: Parameter Updated with a new type successfully.
-
-        :caseautomation: notautomated
+        :expectedresults: Parameter updated with a new type successfully.
 
         :CaseImportance: Critical
         """
+        sc_param = self.sc_params_list.pop()
+        with Session(self.browser):
+            for data in valid_sc_parameters_data():
+                with self.subTest(data):
+                    self.sc_parameters.update(
+                        sc_param.parameter,
+                        self.puppet_class.name,
+                        override=True,
+                        key_type=data['sc_type'],
+                        default_value=data['value'],
+                    )
+                    self.sc_parameters.click(
+                        self.sc_parameters.search_sc_params(
+                            sc_param.parameter, self.puppet_class.name)
+                    )
+                    value = self.sc_parameters.wait_until_element(
+                        locators['sc_parameters.default_value']).text
```

Comment:
We have 5 different types of inputs there, so fetcher there is a really bad idea.
About `\n..`. Yes, in case we have such helper, probably we can make it even more complex to evaluate that statement for one very-very specific case. And it was not about removing, but adding extra line =)